NCR-Corporation/sample-app-burgers
# MARKET-7170 fix burger app addon price calculation

## Description
This PR fixes the issue where total shows up as NaN in check out step.

## Screenshots/Screencasts
NaN

## Testing
Please outline how to test this fix or feature. Include all steps.

1. Go https://burgersdemo.ncrcloud.com/Peachtree-Burger/
2. Add Pretzel without Mustard to cart
3. See total being calculated correctly

## Ticket
MARKET-7170
